projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling>=1.4.0", "jupyterlab>=3.4.7,<4.0.0", "hatch-nodejs-version"]
+requires = ["hatchling>=1.4.0", "jupyterlab==3.6.3", "hatch-nodejs-version"]
```

Comment:
I don't agree with this, it should still be a range

```
>=3.4.7,<=3.6.3
```

Reply:
@DeltaMichael in the npm package we pinned all the versions to 3.6.3 and the builders, as well, I ma not sure whether having lower versions here might cause some problem to not

Reply:
My concern is if someone with an older version of Jupyter is using the extension and then tries to update. What's the user experience in this case? Otherwise, I'm ok with pinning it to 3.6.3 if it's pinned everywhere else.